google/cloud/bigquery/_helpers.py

```diff
@@ -120,11 +121,34 @@ def verify_version(self):
         """
         if self.installed_version < _MIN_BQ_STORAGE_VERSION:
             msg = (
-                "Dependency google-cloud-bigquery-storage is outdated, please upgrade "
-                f"it to version >= {_MIN_BQ_STORAGE_VERSION} (version found: {self.installed_version})."
+                "Dependency google-cloud-bigquery-storage is outdated, "
+                f"please upgrade it to version >= {_MIN_BQ_STORAGE_VERSION} "
+                f"(version found: {self.installed_version})."
             )
             raise LegacyBigQueryStorageError(msg)
 
+    def try_import(self) -> Any:
+        """Tries to import the bigquery_storage module, and returns an
+        error if BigQuery Storage extra is not installed.
+
+        Returns:
+            The ``bigquery_storage`` module.
+
+        Raises:
+            BigQueryStorageNotFoundError:
+                If google-cloud-bigquery-storage is not installed.
+        """
+        try:
+            from google.cloud import bigquery_storage  # type: ignore
+        except ImportError:
+            msg = (
+                "Package google-cloud-bigquery-storage not found. "
+                "Install google-cloud-bigquery-storage version >= "
+                f"{_MIN_BQ_STORAGE_VERSION}."
+            )
+            raise BigQueryStorageNotFoundError(msg)
+        return bigquery_storage
```

Comment:
Let's do the same kind of version checks as we do for pyarrow. We can `raise_if_error: bool = False` like we have there too and remove the redundant `verify_version()` method.

Reply:
Done, change made at #1699